ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -123,10 +128,10 @@ def validate_config(self, config):
 
                 # Check if default groups exist
                 context = {'model': model, 'user': c.user}
-                for group_name in config_obj['default_groups']:
+                for group_ in config_obj['default_groups']:
```

Comment:
@amercader Just to let you know that I reverted the change made in the default groups.